Plots/Satellites/NCL_sat_1.py

```diff
@@ -0,0 +1,191 @@
+
+    minimacoords = []
+
+    # For every common zero in both gradient arrays 
+    for x in commonzeroes:
+
+        xval = x[0]
+        yval = x[1]-1
+
+        # If the gradient value is a "zero", and if the U.data value is less than minPressure
+        if -minPressure <= wrap_U.data[xval][yval] < minPressure:
+
+            coordonmap = coordarr[xval][yval]
+
+            # Transform data points to match globe coordinate scale
+            xcoord = coordonmap[0]
+            ycoord = coordonmap[1]
+
+            minimacoords.append((xcoord, ycoord))
+
+    return minimacoords
+
+###############################################################################
+# Create plot
+
+# Set figure size
+fig = plt.figure(figsize=(8, 8))
+
+# Set global axes with an orthographic projection
+proj = ccrs.Orthographic(central_longitude=270, central_latitude=45)
+ax = plt.axes(projection=proj)
+ax.set_global()
+
+# Add land, coastlines, and ocean features
+ax.add_feature(cfeature.LAND, facecolor='lightgray')
+ax.add_feature(cfeature.COASTLINE, linewidth=.5)
+ax.add_feature(cfeature.OCEAN, facecolor='lightcyan')
+ax.add_feature(cfeature.BORDERS, linewidth=.5)
+ax.add_feature(cfeature.LAKES, facecolor='lightcyan', edgecolor='k', linewidth=.5)
+
+# Make array of the contour levels that will be plotted
+contours = np.arange(948, 1060, 4)
+contours = np.append(contours, 975)
+contours = np.sort(contours)
+
+# Plot contour data
+p = wrap_U.plot.contour(ax=ax,
+                        transform=ccrs.PlateCarree(),
+                        linewidths=0.5,
+                        levels=contours,
+                        cmap='black',
+                        add_labels=False)
+
+# Specify array of contour levels to be labeled- These values were found by setting 'manual'
+# argument in ax.clabel call to 'True' and then hovering mouse over desired location of
+# countour label to find coordinate (which can be found in bottom left of figure window)
+
+# low pressure contour levels- these will be plotted as a subscript to an 'L' symbol
+#lowClevels = [(51.54, 169.59), (74.78, 4.54), (60.12, -57.0)]
+lowClevels = findLocalMinima()
+
+# regular pressure contour levels
+clevels = [(34.63, 176.4), (42.44, -150.46), (28.5, -142.16),
+           (16.32, -134.12), (17.08, -108.90), (15.60, -98.17),
+           (42.19, -108.73), (49.66, -111.25), (41.93, -127.83),
+           (25.64, -92.49), (29.08, -77.29), (16.42, -77.04),
+           (57.59, -95.93), (84.47, -156.05), (82.52, -17.83),
+           (41.99, -76.3), (41.45, -48.89), (37.55, -33.43),
+           (17.17, -46.98), (63.67, 1.79), (67.05, -58.78),
+           (53.68, -44.78), (53.71, -69.69), (52.22, -78.02),
+           (44.33, -16.91), (35.17, -95.72), (73.62, -102.69)]
+
+# Transform the low pressure contour coordinates from geographic to projected
+lowclevelpoints = proj.transform_points(ccrs.Geodetic(), np.array([x[1] for x in lowClevels]), np.array([x[0] for x in lowClevels]))
```

Comment:
You may have fixed the flipped lon/ lat coordinates here by calling `x[1]` as the x coordinates and `x[0]` as the y coordinates. You will need to change this line if you fix `makeCoordsArr`